### ISSUE TITLE
[Issue]: Unable to quit streaming session on Logitech G Cloud through hotkeys

### ISSUE THREAD

**Issue by cobra872** [moonlight-stream/moonlight-android#1227]
### Describe the bug

The gamepad hotkey to quit a streaming session is start+select+l1+r1. However, this does not work on the Logitech G Cloud. Start+select takes a screenshot on the G Cloud, and there is no way to remap the buttons. 

Obviously, this could be seen as a problem on Logitech’s side, but I think there should be alternate options to quit streaming sessions for people that may have this problem.

### Steps to reproduce

Any time start+select is pressed

### Affected games

All game sessions

### Other Moonlight clients

PC

### Moonlight adjusted settings

Yes

### Moonlight adjusted settings (please complete the following information)

Display options that don’t correlate to the gamepad input issue

### Moonlight default settings

Yes

### Gamepad-related connection issue

No

### Gamepad-related input issue

No

### Gamepad-related streaming issue

Yes

### Android version

Android 11

### Device model

Logitech G Cloud

### Server PC OS version

Windows 11

### Server PC GeForce Experience version

3.27.0.112

### Server PC Nvidia GPU driver version

536.23

### Server PC antivirus and firewall software

Malwarebytes Premium

**Comment by provencher** [comment]
Hey I ran into this issue as well. Still looking for a workaround.

**Comment by cobra872** [comment]
> Hey I ran into this issue as well. Still looking for a workaround.

Yeah, but I actually found a solution not too long ago. If you just swipe left from the edge of the screen it ends the session and brings you back to the menu. It’s a lot easier too. Only problem is you can do this by accident from time to time while using the touch screen as a mouse cursor.
